themes/ambiance/lxqt-panel.qss

```diff
@@ -311,6 +311,134 @@ QCalendarWidget QWidget {
     color: palette(highlighted-text);
 }
 
+/*
+ *Fancy Menu
+*/
+
+#FancyMenu {
+    margin: 2px;
+    padding: 1px;
+    border-radius: 4px;
+    qproperty-icon: url(mainmenu.svg);
+    color: #f2f1f0;
+    border: none;
+}
+
+#FancyMenu:pressed {
+    padding: 4px;
+}
+
+LXQtFancyMenuWindow{
+    background:  #3c3b37;
+}
+
+#FancyMenu #AppView {
+    background-color: #504f49;
+    border: 1px solid #5e5e5e;
+}
+
+#FancyMenu #CategoryView {
+    background-color: #32312e;
+}
+
+#FancyMenu QListView::icon {
+    background-color: transparent;
+    border: 10px solid transparent;
+    border-top-width: 0px;
+    border-bottom-width: 0px;
+    border-right-width: 0px;
+}
+
+#FancyMenu QListView::item {
+    color: #f2f1f0;
+    padding: 5px 0px 5px 14px;
+}
```

Comment:
Shift to right is still visible with some Kvantum style themes -- it's only by chance that it may not be visible with Fusion because padding is changed below.

Now, there are two options. If you want to keep `padding: 5px 0px 5px 14px;` (which looks OK), make it as

```css
#FancyMenu QListView::item {
    color: #f2f1f0;
    padding: 5px 0px 5px 14px;
    border: 0px;
}
```

and *remove `padding: 0px;` from the following, related blocks*.

But if you don't wan the padding, just do as

```css
#FancyMenu QListView::item {
    color: #f2f1f0;
    padding: 0px;
    border: 0px;
}
```

Reply:
Done with padding.